src/commands/useful/append.js

```diff
@@ -0,0 +1,43 @@
+import Promise from 'bluebird';
+import sentry from '../../sentry';
+import commands from '../index';
+import { callCmd } from '../../index';
+
+function parseMessage(bot, content, msg) {
+  let command = content.toLowerCase().split(' ')[0].substring(1);
+  let suffix = content.substring(command.length + 2);
+  let cmd = commands[command];
+
+  if (cmd) {
+    callCmd(cmd, command, bot, msg, suffix);
+  }
+  return;
+}
+
+
+function append(bot, msg, suffix) {
+  if (!suffix) {
+    bot.sendMessage(msg.channel, 'Usage: **`!!`** `text`');
+    return;
+  }
+
+  Promise.resolve(msg.channel.messages.getAll('author', msg.author))
+    .then(messages => {
+      messages.pop();
+      if (!messages[0]) {
+        return;
+      }
+
+      let newCmd = messages.pop().content.trim().concat(' ').concat(suffix);
+      bot.sendMessage(msg.channel, newCmd);
+      parseMessage(bot, newCmd, msg);
```

Comment:
This way I was also allowing it to be used for regular text too, things that are not commands. This also allowed the bot to show the full command it is executing when using after appening something. Do you think it should just be `parseMessage()`?